multiplexer.go

```diff
@@ -57,23 +64,34 @@ func (m *multiplexer) Add(matcher TopicMatcher, handler interface{}) error {
 	return nil
 }
 
-func (m *multiplexer) callback(topic Topic, data map[string]interface{}) {
+// Add another topic matcher and handler to the multiplexer.
```

Comment:
nit - comment should start with // Unsubscribe .....

plus it looks like cut and paste error

and comment on previous method should start with // AddMatch .....